content/recipes/authoring/kata-snippets/description.md

```diff
@@ -5,6 +5,54 @@ prev: /recipes/authoring/kata-snippets/
+- Descriptions should provide **examples of inputs and outputs for some representative scenarios**. Provide as many examples as you think necessary.
+  - **Avoid things like _"See sample tests for more examples"_**. To see sample tests, users have to enter the kata trainer, what can mark the kata as unfinished in their profile.
+  - **Use images when helpful.** Currently, Codewars does not allow to attach files to a kata, so image has to be hosted externally, or HTML image with Base-64 encoded content, preferrably SVG, can be used. Make sure that image does not break layout of the kata page.
+- **Describe or hint the difficulty.** It's a very frustrating experience when a user comes up with solution returning correct answers, only to find out that it times out upon submission. Provide information about input ranges, volume of tests, expected complexity class, etc. 
+
+
+## Formatting
+
+- **Use [Markdown capabilities](/references/markdown/)** to format and structure descriptions appropriately. You can use headers and lines to introduce sections, lists and tables to collect various items, links to provide references, code markup to introduce code inline or in blocks, and basic text formatting for styling. Do not abuse it though, avoid visual noise and excessive formatting.
+- **Use proper header levels.** Do not use `# Title` to introduce sections, consider `## Second level` or `### Third level` headers for organization.
+- **Use [Sequential Code Blocks](/references/markdown/extensions/#sequential-code-blocks) for blocks of code, examples, etc.
+- **Avoid descriptions mentioning many languages.** Use [Language Conditional Rendering](/references/markdown/extensions/#conditional-rendering) for language specific paragraphs and sections. Do not stuff many languages into a single sentence or paragraph (_"Return `null` for Java/C#/Scala, `None` for Python, `Nothing` for Haskell, `NULL` for C/C++, `nil` for Swift/Ruby"_).
+- **Avoid HTML.** Interactions between embedded HTML and Markdown or theming are not well defined and can break in unexpected way. HTML used for styling can give wrong results after CSS is updated or can not work well with visual themes (for example dark or light mode). HTML **can** be used as long as there's a valid purpose and the author is aware of the downsides.
+
+
+## Other
+
+- **Description can contain links and references** to external sources of knowledge, articles, honts, explanations, papers.
+- **Description can contain links to related kata**: other variants of the task, difficulty levels, challenge versions. Links should go in both directions, for example easier kata should link to difficult one, and vice versa.
```

Comment:
```suggestion
- **Description can contain links to related kata**: other variants of the task, difficulty levels and challenge versions.
```

Reply:
I wanted to be explicit about links going in both directions, because it sometimes happened that I wanted to compare two versions of a problem (easier and harder), but the one I was looking at was missing the link.

Reply:
Sure, feel free to keep it in.